src/test/java/teammates/test/cases/ui/browsertests/AdminAccountManagementPageUiTest.java

```diff
@@ -34,20 +34,22 @@
     
     @BeforeClass
     public static void classSetup() throws Exception {
-        printTestClassHeader();
+        // Temporarily disable unit test for Account Management Page.
```

Comment:
@bamboo3250, I think the comment should be explaining why the test is disabled. 

Instead of disabling everything, we should still have the test for the current functionality , where only one instructor appears on the account management page, can still work.